src/main/java/com/adorsys/ssi/sdjwt/SdJws.java

```diff
@@ -122,6 +123,34 @@ public void verifyNotBeforeClaim() throws SdJwtVerificationException {
         }
     }
 
+    /**
+     * Verifies that SD-JWT was issued by one of the provided issuers
+     * @param issuers List of trusted issuers to lowercase
+     */
```

Comment:
I sense that what your aim is a case-insensitive check... You might add that in the comment and lowercase both the issuer claim and the provided list of issuers yourself.